vassal-app/src/main/java/VASSAL/build/module/map/BoardPicker.java

```diff
@@ -172,9 +175,7 @@ public void addTo(Buildable b) {
     for (Board board : possibleBoards) {
       board.setMap(map);
     }
-    if (b instanceof Translatable) {
```

Comment:
Ditto here---isn't this going to cause us a problem if `b` isn't a `Translatable`?

Reply:
Same again, fixed.